### PR TITLE
Fixes #RHINENG-5028 - support multiple kafka servers

### PR DESCRIPTION
**Pull request by upadhyeammit**
## Why do we need this change? :thought_balloon:

The platform now has multiple kafka servers, the change is to support new additional servers/all servers

## Documentation update? :memo:

- [ ] Yes
- [ ] No

## Security Checklist :lock:

Upon raising this PR please go through [RedHatInsights/secure-coding-checklist](https://github.com/RedHatInsights/secure-coding-checklist)

## :guardsman: Checklist :dart:

- [ ] Bugfix
- [ ] New Feature
- [ ] Refactor
- [ ] Unittests Added
- [ ] DRY code
- [ ] Dependency Added
- [ ] DB Migration Added

## Additional :mega:

Feel free to add any other relevant details such as __links, notes, screenshots__, here.

### CODE REVIEW

**Comment by upadhyeammit** [comment]
We had group.id and enable.auto.commit for producer as well, as per the logs its not required. I can also see one more ERROR of CLOUDWATCH, however not sure if that should be fixed in this pull request,
~~~
2023-12-07 11:46:38,372 - ERROR - commence_cw_log_streaming - CLOUDWATCH LOGGING - Insufficient constant values
2023-12-07 11:46:38,374 - INFO - run - PROCESSING ENGINE RESULTS - Processor is running. Awaiting msgs.
%4|1701949598.375|CONFWARN|rdkafka#producer-2| [thrd:app]: Configuration property group.id is a consumer property and will be ignored by this producer instance
%4|1701949598.375|CONFWARN|rdkafka#producer-2| [thrd:app]: Configuration property enable.auto.commit is a consumer property and will be ignored by this producer instance
~~~

File: ros/lib/produce.py [hunk 1]
```diff
@@ -1,13 +1,7 @@
...
 
 def init_producer():
-    connection_object = {
-            'group.id': GROUP_ID,
-            'bootstrap.servers': INSIGHTS_KAFKA_ADDRESS,
-            'enable.auto.commit': False
-    }
-    producer = Producer(kafka_auth_config(connection_object))
-
+    producer = Producer(kafka_auth_config())
```
Comment (r14chandra) [inline ros/lib/produce.py:6]: Any particular reason for omitting `group.id` and `enable.auto.commit` attributes from here?

**Review by r14chandra — CHANGES_REQUESTED** [review]

File: ros/lib/produce.py (hunk 1 above)
Comment (r14chandra) [reply ros/lib/produce.py:6]: Okay, I missed your above comment. It makes sense to remove it.

**Review by r14chandra — APPROVED** [review]
LGTM!

**Comment by upadhyeammit** [comment]
Thanks Rohini for the review! Merging this now!

**Pull request merged**